doc/command_ref.rst

```diff
@@ -1091,10 +1095,9 @@ Display duplicated packages::
 
     dnf repoquery --duplicates
 
-Remove older versions of duplicated packages (an equivalent of yum's `package-cleanup --cleandups`)::
-
-    dnf remove --duplicates
+Display source packages that requires for build a <provide>::
```

Comment:
`Display source packages that requires a <provide> for a build` sounds a bit better, IMHO.